frontend/src/components/Tree/Tree.jsx

```diff
@@ -167,7 +179,35 @@ function renderTree(
     .attr("opacity", (d) => {
       return d.data.opacity;
     })
-    .text((node) => node.data.name);
+    //.append("title")
+    .text((node) => {
+      console.log(node.data.name);
+      if (hoverText === node.data.name) return node.data.name;
+
+      const spaceIndex = node.data.name.indexOf(" ");
+      if (spaceIndex !== -1) {
+        if (spaceIndex > TreeConf.LABEL_MAX_LENGTH + 3) {
```

Comment:
I thought it would give them a better idea of the content of a given node ukwim? that way including first word before space would give more info about the node. what do you think?

Reply:
Oh, I thought the opposite, I was thinking that showing them more characters is better so we don't truncate at the first space but we keep showing more of the label if we can

Reply:
I see what you mean, I will fix that up

Reply:
fixed